### ISSUE TITLE
Extension methods on BlockContext break compatibility to Squeak-trunk

### ISSUE THREAD

**Issue by fniephaus** [dalehenrich/metacello-work#434]
`BlockContext` has been replaced in `Squeak-trunk` with `Context`.
This breaks Metacello compatibility to `Squeak-trunk`.
Unfortunately, I don't have time to look into this at the moment.

**Comment by dalehenrich** [comment]
I don't have time to dig into this either :( 

 [There a handful of messages added to BlockContext](https://github.com/dalehenrich/metacello-work/tree/master/repository/Metacello-Platform.squeak.package/BlockContext.extension/instance) ... should be straightforward ...

**Comment by fniephaus** [comment]
No problem, just wanted to leave this issue here, so we don't forgot. In the meantime, I understand @codezeilen will soon re-add the classes to Squeak as part of a deprecation package. Nonetheless, we will need to update Metacello soon...

/cc @krono

**Comment by dalehenrich** [comment]
Makes sense ... I'm deep into work for the upcoming 3.4 work, if or when I come up for error, Metacello will be the next thing that I tackle, since Metacello is/will be ported to the base GemStone system for 3.4 as well :)

**Comment by hhzl** [comment]
In the meantime a trunk update has been made which reintroduces the missing classes under the 'Deprecated6.0' category.

Metacello loads fine again in the latest Squeak6.0a trunk version.
